src/Export/Adapters/ExportItemAdapter.php

```diff
@@ -136,27 +141,37 @@ public function adaptProduct(Item $item, ProductEntity $product): ?Item
             $item->addUsergroup($userGroup);
         }
 
+        foreach ($this->adapterFactory->getOptionsAdapter()->getOptionAttributes($product) as $attribute) {
+            $item->addMergedAttribute($attribute);
+        }
+
         return $item;
     }
 
     public function adaptVariant(Item $item, ProductEntity $product): ?Item
     {
         if ($this->eventDispatcher) {
-            $this->eventDispatcher->dispatch(new BeforeVariantAdaptEvent($product, $item), BeforeVariantAdaptEvent::NAME);
+            $this->eventDispatcher->dispatch(
+                new BeforeVariantAdaptEvent($product, $item),
+                BeforeVariantAdaptEvent::NAME,
+            );
         }
 
         try {
             foreach ($this->adapterFactory->getOrderNumbersAdapter()->adapt($product) as $orderNumber) {
                 $item->addOrdernumber($orderNumber);
             }
 
-            foreach ($this->adapterFactory->getAttributeAdapter()->adapt($product) as $attribute) {
```

Comment:
The attributes adapter will still be needed, the issue is just about the options.